types/user.go

```diff
@@ -23,6 +24,11 @@ func (u *User) Validate() error {
 	return nil
 }
 
+// URIPath returns the path component of a User URI.
```

Comment:
I'm not so sure we want to allow creating Users via RBAC just yet.

This would mean people would be storing raw passwords in CM, which isn't very secure.

I want to have a discussion about how we can support this, but I think we'll have to change some things about User first.

Reply:
Okay, I've changed this and addressed the other feedback. I've also made a note that `wrapped-json` is unsupported for `User` because of this!